src/skin/skinloader.cpp

```diff
@@ -65,6 +65,7 @@ QPixmap SkinLoader::getSkinPreview(const QString& skinName, const QString& schem
         preview.load(getSkinPath(skinName) + "/skin_preview.png");
     }
     if (!preview.isNull()){
+        preview = preview.scaled(QSize(640, 360), Qt::KeepAspectRatio, Qt::SmoothTransformation);
         return preview;
     }
     preview.load(":/images/skin_preview_placeholder.png");
```

Comment:
```suggestion
    if (preview.isNull()) {
        preview.load(":/images/skin_preview_placeholder.png");
    }
    preview = preview.scaled(QSize(640, 360), Qt::KeepAspectRatio, Qt::SmoothTransformation);
```
..to include the placeholder.
if we then create all previews with a ratio of 16:9 all options below have a fixed position.

Reply:
ping